static/app/views/replays/detail/console/consoleMessage.tsx

```diff
@@ -123,8 +129,10 @@ const ConsoleTimestamp = styled(Common)<{isLast: boolean; level: string}>`
   cursor: pointer;
 `;
 
-const Icon = styled(Common)`
+const Icon = styled(Common)<{isActive: boolean}>`
   padding: ${space(0.5)} ${space(1)};
+  border-left: 4px solid
```

Comment:
An issue with using a border for the active state is that the bottom border is affected when it is not active:

![image](https://user-images.githubusercontent.com/79684/170143461-7303af9e-3350-4119-b62e-66603eb17d0b.png)


(Also the row can be have different bg colors depending on log level).

Reply:
Any ideas on alternatives? I initially thought the background color could be changed but I'd worry that interferes with the current styling of the row.